Yoast/wordpress-seo
# Add passive voice assessment for Slovak

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the passive voice assessment for Slovak.
* Adds the assessment for detecting passive voice in Slovak.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Enable the `SLOVAK_SUPPORT` feature by adding `define( 'YOAST_SEO_SLOVAK_SUPPORT', true );` in `basic-wordpress-config.php` file
* Set your website to Slovak.
* Test this functionality in Yoast SEO Free.

* Create a new empty post and add these sentences to your post:
`Topánky sú leštené`
`Bola mačka adoptované alebo bola kúpená?`
`Stôl je vyčistený.`
`Jedlo bude včas uvarené.`
`Jeho žiadosť bola zamietnutá.`
`Chce byť skontrolovaný včas.`

* Check that the passive voice bullet turns red and all sentences are highlighted.

* Add these sentences and check that that they do not get highlighted:
`Túto úlohu študenti vykonali včera.`
`Jedlo je hotové, stôl je pripravený.`

* Disable the `SLOVAK_SUPPORT` by adding this `define( 'YOAST_SEO_SLOVAK_SUPPORT', false );` to `basic-wordpress-config.php` file
* Save the changes in the post
* Reload your page
* Make sure that the assessment doesn't appear in the metabox

### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=99&modal=detail&selectedIssue=LINGO-894
